core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/ExoPlayerForwarder.java

```diff
@@ -43,6 +44,10 @@ public AnalyticsListener analyticsListener() {
         return analyticsListener;
     }
 
+    public NoPlayer.AdvertListener advertListener() {
```

Comment:
Shouldn't this be `@Nullable` then? (or is `Optional` preferred in the project?)

Reply:
Ah yes 😄